tests/measure/editors/test_execution_editor.py

```diff
@@ -35,6 +33,11 @@
 def task():
     """Create a basic task hierarchy for testing.
 
+    Root
+      - SimpleTask: parallel=False, wait=False
+      - ComplexTask: parallel=False, wait=True
+        - SimpleTask: parallel=True, wait= False
```

Comment:
This simply document the structure of the returned task and the parallel/wait settings the indentation indicate the parent/child relation but I can increase indentation to 4 spaces